packages/valory/skills/abstract_round_abci/behaviour_utils.py

```diff
@@ -369,85 +374,77 @@ def __handle_stop_iteration(self) -> None:
         self.__state = self.AsyncState.READY
 
 
-def _check_ipfs_enabled(fn: Callable) -> Callable:
-    """Decorator that raises error if IPFS is not enabled."""
-
-    @wraps(fn)
-    def wrapper(*args: Any, **kwargs: Any) -> Any:
-        """The wrap that checks and raises the error."""
-        ipfs_enabled = args[0].ipfs_enabled
-
-        if not ipfs_enabled:  # pragma: no cover
-            raise ValueError(
-                "Trying to perform an IPFS operation, but IPFS has not been enabled! "
-                "Please set `ipfs_domain_name` configuration."
-            )
-
-        return fn(*args, **kwargs)
-
-    return wrapper
-
-
 class IPFSBehaviour(SimpleBehaviour, ABC):
     """Behaviour for interactions with IPFS."""
 
     def __init__(self, **kwargs: Any):
         """Initialize an `IPFSBehaviour`."""
         super().__init__(**kwargs)
-        self.ipfs_enabled = False
-        # If params are not found `AttributeError` will be raised. This is fine, because something will have gone wrong.
-        # If `ipfs_domain_name` is not specified for the skill, then we get a `None` default.
-        # Therefore, `IPFSBehaviour` will be disabled.
-        domain = getattr(self.params, "ipfs_domain_name", None)  # pylint: disable=E1101
```

Comment:
Please update the models.py and remove the attribute there as it is no longer used anywhere now afaik